dts/bindings/usb-c/usb-c-connector.yaml

```diff
@@ -0,0 +1,173 @@
+# Copyright 2022 The Chromium OS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    A USB Type-C connector node represents a physical USB Type-C connector.
+    It should be a child of a USB-C interface controller or a separate node
+    when it is attached to both MUX and USB-C interface controller.
+
+    This is based on Linux, documentation:
+      https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/devicetree/bindings/connector/usb-connector.yaml?h=v5.19&id=3d7cb6b04c3f3115719235cc6866b10326de34cd
```

Comment:
The link, also in commit message, should be https://www.kernel.org/doc/Documentation/devicetree/bindings/connector/usb-connector.yaml

Reply:
git grep "based on Linux" dts/

Reply:
Done